change/@ni-eslint-config-angular-7bcf1406-f6fc-4336-a2ff-a2fbb32de903.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Just a note that we decided not to complete this PR until key clients are ready to uptake it.

Reply:
There's really that much concern? Shouldn't this be much easier/faster than the Material upgrade?

Reply:
We're just saying that it's more expedient to wait for client teams to complete their Angular 16 upgrade testing before we complete this PR. We could complete it earlier but didn't see any benefit to doing so, just churn of Renovate PRs that would have to be ignored / rejected until the Angular upgrade is ready.

Reply:
Ohh, I misunderstood. Thanks for clarifying.